GooglePlayInstant/Samples/TestApp/Scripts/TestApp.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using UnityEngine;
+
+namespace GooglePlayInstant.Samples.TestApp
+{
+    /// <summary>
+    /// Tests instant app plugin features through button clicks
+    /// </summary>
+    public class TestApp : MonoBehaviour
+    {
+        private const string CookiePrefix = "test-cookie";
+        private string _storedCookie;
+
+        /// <summary>
+        /// Sets the instant app cookie to a unique string
+        /// </summary>
+        public void WriteCookie()

```

Comment:
I'm failing to comment on the previous thread, but regarding "indicate that these functions were referenced by a button in the scene" I'm definitely not opposed to that!

My point was that the abbreviations and underscore were not consistent with the codebase.

So ButtonEventWriteCookie or ButtonHandlerWriteCookie or similar would be fine

Reply:
Cool, I'll rename them with a ButtonEvent prefix.